.circleci/config.yml

```diff
@@ -187,20 +187,20 @@ jobs:
           steps:
             - run: ./diff.sh content-service || (echo "no diff" && circleci-agent step halt)
       # Download and cache dependencies
-      - restore_cache:
-          keys:
-          - content-service-{{ checksum "content-service/package.json" }}
-          # fallback to using the latest cache if no exact match is found
-          - content-service-
+      # - restore_cache:
```

Comment:
Content service is throwing the same error as creator node after node version upgrade on circle so disabling for now